src/librustc_mir/interpret/intrinsics.rs

```diff
@@ -150,6 +150,9 @@ impl<'a, 'mir, 'tcx, M: Machine<'a, 'mir, 'tcx>> EvalContext<'a, 'mir, 'tcx, M>
                 }
                 self.write_scalar(val, dest)?;
             }
+            "init" => {
+                self.force_allocation(dest)?;
```

Comment:
You can just copy the implementation from https://github.com/solson/miri/blob/master/src/intrinsic.rs#L238

Your current implementation is a nop, it literally does nothing ;)

Reply:
Dang, that was awfully naive of me ha. Done.

Side question: is there any reason that entire file isn't just copied into here?